recipe/meta.yaml

```diff
@@ -271,20 +276,33 @@ outputs:
         # decide on the cuda variant
         # https://github.com/conda-forge/tensorflow-feedstock/issues/162
         - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-        - bazel 4.2.2
+        - bazel >=5.2.0
+        - libabseil
+        - grpc-cpp
+        - numpy 1.20.*  # [py<310]
+        - numpy 1.21.*  # [py>=310]
+        - openssl
+        - libprotobuf
+        - {{ pin_subpackage('tensorflow-base', exact=True) }}
       host:
         - python
         - pip
+        - numpy 1.20.*  # [py<310]
+        - numpy 1.21.*  # [py>=310]
```

Comment:
I prefer that this comes from the pinning (also above twice). The lower bound can still be enforced on a separate line.
```suggestion
        - numpy
        - numpy >=1.20
```

Reply:
> Does this actually work?

Yes it does, because there's no numpy 1.20 for python 3.10. It would anyway be time to bump our minimum to 1.20 according to NEP29 (which would allow dropping the second line), but that's a separate topic.

Reply:
I unfortunately do not this that the proposed syntax works. I owuld leave the original syntax
```
- numpy   1.20  # [py<310]
- numpy   1.21  # [py>=310]
```

Reply:
Yeah, I might have gotten something mixed up be right, but in any case, the following will allow us to only depend on unpinned numpy: github.com/conda-forge/conda-forge-pinning-feedstock/pull/3420

Reply:
lets not depend on this.